### ISSUE TITLE
Autofix applying wrong element

### ISSUE THREAD

**Issue by msorens** [semgrep/semgrep#1946]
**Describe the bug**
The variable descriptor is being used instead of the variable itself.
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/6817500/97812194-8d558000-1c34-11eb-980a-50b375c6986d.png">

**To Reproduce**
https://semgrep.dev/s/6WXK/

Note: This sample actually contains 3 bugs; filing separately.

**Expected behavior**
Should be this:
```
return events.find(event => event.entity_name === entityName).color || {};
```
and not this:
```
return events.find(const => event.entity_name === entityName).color || {};
```

**What is the priority of the bug to you?**
Is this a P0 (blocking your adoption of Semgrep or workflow), P1 (important to fix or quite annoying), P2 (regular bug that should get fixed)?
P2

**Environment**
semgrep 0.29.0

**Comment by mjambon** [comment]
Thank you @msorens!

**Comment by msorens** [comment]
This is fixed in production as far as I can tell; OK to close. 😁

**Comment by ievans** [comment]
Confirmed fixed in 0.77 or earlier!
